### ISSUE TITLE
Any support for Partials?

### ISSUE THREAD

**Issue by drone1** [hadijaveed/reactive-handlebars#2]
Hey again.

While you make it quite clear this is more for a simple one-pager type app, that is pretty much what I'm using it for and I'm having a hard time figuring out a way to actually make use of it, since changing a template var causes the entire template to re-render, and there doesn't seem to be support for reactive partials.

I've got a simple page that plays video. Below the video there are some tabs for the user to click through. However, the way it's currently implemented, changing my tab template variable just re-renders the entire page, which of course interrupts the video.

Any way you're aware of that I can re-render only my tab area, with partials or something like this? Otherwise I guess I'll just have to always render all tabs and then show/hide using Javascript. :/

Thanks for any help.

**Comment by hadijaveed** [comment]
@drone1 yeah this library don't support partials at the moment. Since it was a small library to support HBS templates. I can work on it though.

**Comment by drone1** [comment]
I bit the bullet and used jquery. Would love support for partial reactive
rendering!
